src/v3/src/transformer/authJs.ts

```diff
@@ -10,43 +10,55 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import { IdxFeature } from '@okta/okta-auth-js';
-import { FormBag, IdxTransactionWithNextStep, WidgetProps } from 'src/types';
+import { IdxFeature, IdxTransaction } from '@okta/okta-auth-js';
+import { FormBag, WidgetProps } from 'src/types';
 
 import { AUTHENTICATOR_KEY, IDX_STEP, STEPS_MISSING_RELATES_TO } from '../constants';
 import { hasMinAuthenticatorOptions } from '../util';
 import { transformInputs } from './field';
 import { getButtonControls } from './getButtonControls';
 import TransformerMap from './idxTransformerMapping';
 import { transformCustomMessages } from './messages/transformCustomMessages';
-import { remediationContainsStep } from './utils';
 
-export default (transaction: IdxTransactionWithNextStep, widgetProps: WidgetProps): FormBag => {
-  const { nextStep, context } = transaction;
+type Options = {
+  transaction: IdxTransaction;
+  prevTransaction: IdxTransaction | undefined;
+  step?: string;
+  widgetProps: WidgetProps;
+};
```

Comment:
`TransformerOptions` ?

Also, shouldn't this be added to the `types` folder and exported for re-use?

Reply:
it's just an internal type, not exported